packages/page-staking/src/Actions/partials/Bond.tsx

```diff
@@ -120,12 +120,13 @@ function Bond ({ className = '', isNominating, minNominated, minNominatorBond, m
                 <p>{t('To ensure optimal fund security using the same stash/controller is strongly discouraged, but not forbidden.')}</p>
               </>
             )
-            : (
-              <>
```

Comment:
Can we just put comment 
```
// Uncomment when aleph-node contains pallet proxy
```

Reply:
I changed the solution to hide and show it base on pallet proxy availability